MetaMask/metamask-extension
# feat: Add sepolia and update ppom

## **Description**
After last ppom version has been released v1.4.2, we can now use Blockaid in Sepolia. We should add the necessary changes to support it in Extension.

Extra note: there's an[ upcoming change in geth](https://github.com/ethereum/go-ethereum/releases/tag/v1.13.14), which might affect[ eth_creatAccessList](https://github.com/ethereum/go-ethereum/pull/28846) so we need to test this in Sepolia before this is implemented in Mainnet

[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/23298?quickstart=1)

## **Related issues**

Fixes: #23272 

## **Manual testing steps**

1. Build MM and launch
2. Change change to sepolia
3. Enable Blockaid
4. Do a PPOM transaction on testdapp
5. Notice Blockaid validation is not done
6. Checkout this branch
7. Repeat step 1-4
8. Notice Blockaid validation is now down and errors shown

## **Screenshots/Recordings**



### **Before**

https://github.com/MetaMask/metamask-extension/assets/44811/f9c9b126-8692-4b8e-95ce-8649ae20201a

### **After**

https://github.com/MetaMask/metamask-extension/assets/44811/dcdacd9a-a2ef-43f1-ac1c-f90f2296e34c

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
